src/wwElement.vue

```diff
@@ -73,23 +81,28 @@ export default {
             if (!this.content) return 'text';
             return this.content.type === 'decimal' ? 'number' : this.content.type;
         },
-        step() {
-            if (!this.content) return '1';
-            return this.content.type === 'decimal' ? this.content.precision : '1';
-        },
     },
     watch: {
-        'content.value'(value) {
-            this.value = value;
+        'content.value'(newValue, OldValue) {
+            if (this.content.type === 'decimal') newValue = this.formatValue(newValue);
+            if (newValue === OldValue) return;
```

Comment:
même remarque ici, on devrait comparer à ce qu'on va set, donc à this.value plutôt qu'à old